crates/holochain/src/conductor/interface/websocket.rs

```diff
@@ -720,7 +720,8 @@ pub mod test {
         // Get state
         let expected = {
             let source_chain = SourceChainBuf::new(cell_env.clone().into()).unwrap();
-            source_chain.dump_as_json().await.unwrap()
+            let r = source_chain.dump_as_json().await.unwrap();
+            serde_json::to_string_pretty(&r).unwrap()
```

Comment:
The assert below on line 734  is failing. Looks like you're going to have to change this test a bit more